src/api/python/cvc5.pxi

```diff
@@ -504,15 +568,16 @@ cdef class Op:
 
     def getNumIndices(self):
         """
-            :return: number of indices of this op.
+            :return: The number of indices of this op.
         """
         return self.cop.getNumIndices()
 
     def __getitem__(self, i):
         """
-            Get the index at position i.
-            :param i: the position of the index to return
-            :return: the index at position i
+            Get the index at position ``i``.
```

Comment:
`__getitem__` is a special member function and not documented by default. It must be added explicitly in `op.rst`, for example with `:special-members`.

Reply:
Right, do we want to add it?

Reply:
Yes, I'd prefer to add this (here and in all other places as well). I'm not so sure about `__str__()`, but maybe we should just show everything that is there?

Reply:
I will not add this in this PR, but separately.

Reply:
see #8513.